cmd/mockserver/server_test.go

```diff
@@ -98,7 +98,7 @@ func client(t *testing.T, port string, wg *sync.WaitGroup) {
 	defer wg.Done()
 	conn, err := net.Dial("tcp", ":"+port)
 	if err != nil {
-		t.Fatal("Could not connect to server:", err)
```

Comment:
Isn't this in the main goroutine and therefore safe?

Reply:
It's called with `go client` in `TestServer`

Reply:
ah, right, never mind... The diff collapsed part of the code, and it seemed like this was part of TestServer function. I should have looked more closely.